wolfssl/wolfcrypt/asn_public.h

```diff
@@ -810,6 +829,20 @@ WOLFSSL_API int wc_Dilithium_PrivateKeyToDer(dilithium_key* key, byte* output,
                                              word32 inLen);
 WOLFSSL_API int wc_Dilithium_PublicKeyToDer(dilithium_key* key, byte* output,
                                             word32 inLen, int withAlg);
+
+WOLFSSL_API int wc_Sphincs_PrivateKeyDecode(const byte* input,
```

Comment:
These are in `sphincs.c`. Should it go into `sphincs.h`? By the way, I like them being in `sphincs.c`.